lib/test_bot.rb

```diff
@@ -57,7 +57,7 @@ def run!(args)
       ENV["HOMEBREW_NO_EMOJI"] = "1"
       ENV["HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK"] = "1"
       ENV["HOMEBREW_FAIL_LOG_LINES"] = "150"
-      ENV["HOMEBREW_CURL_PATH"] = "/usr/bin/curl"
+      ENV["HOMEBREW_CURL_PATH"] = "/usr/bin/curl" if ENV["HOMEBREW_CURL_PATH"].nil?
```

Comment:
We should just set this to the value we want instead of allowing it to be configured from the environment. Something like
```ruby
ENV["HOMEBREW_CURL_PATH"] = ENV["HOMEBREW_ON_DEBIAN7"].present? ? "/opt/bin/curl" : "/usr/bin/curl"
```
will probably do.